test/tree.js

```diff
@@ -0,0 +1,46 @@
+const { Tree } = require('./utils/tree')
+const { TernarySearchTree } = require('../lib/core/tree')
+const assert = require('assert')
+const { test } = require('tap')
+
+function generateAsciiString (length) {
```

Comment:
I think we need/can to support any byte character.

Reply:
I was able to test it correctly using tree.

Reply:
I don't think tree is a good reference. Why does `toString` fail?